.pre-commit-config.yaml

```diff
@@ -42,23 +42,37 @@ repos:
       - id: prettier
         args: [--cache-location=.prettier_cache/cache]
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.14.1
+    rev: v1.15.0
     hooks:
       - id: mypy
-        # Copied from setup.cfg
-        exclude: "properties|asv_bench"
+        files: "^xarray"
+        exclude: "^xarray/util/generate_.*\\.py$"
```

Comment:
Can we keep `# Copied from pyproject.toml`? Ideally this would not be on different settings.

But `pre-commit` oddities are slowing you down, it might be OK fine to turn this off / push to another PR

Reply:
Thanks! Other PRs welcome!